### DISCUSSION TITLE
blog/fix-mongoose-cannot-overwrite-model-once-compiled-error

### DISCUSSION THREAD

**Discussion by giscus** [AshikNesin/nesinio-comments#10] [Announcements]
# blog/fix-mongoose-cannot-overwrite-model-once-compiled-error

https://nesin.io/blog/fix-mongoose-cannot-overwrite-model-once-compiled-error

**Comment by webfrank** [comment]
Genius!!!

**Comment by saifullah0317** [comment]
But then there is an issue "TypeError: _Schema__WEBPACK_IMPORTED_MODULE_0__.Task is not a constructor"

**Reply by AshikNesin** [reply]
Hi @saifullah0317,

That appears to be a problem specific to the project. Can you double-check if you have defined the schema correctly?
